src/types/auth.types.ts

```diff
@@ -0,0 +1,31 @@
+// Request shape when loggin in as a developer (NOT IN PRODUCTION)
+export type DeveloperAuth = {
+  email: string
+  externalId: string
+}
+
+// Shape of resposne when requesting JWT
+export type Token = {
+  accessToken: string
```

Comment:
Do we have a way to get a refresh token via the api right now that isn't cookies? Idk if we do?

Reply:
Thats fair - ideally for MVP id like a way to get tokens via API, but that can be an additional feature/story instead of building it into this already massive changeset